js/src/launchAsync.ts

```diff
@@ -21,6 +21,7 @@ type Message = {
     disableGrammar?: boolean;
     disableTranslation?: boolean;
     disableLanguageDetection?: boolean;
+    disablePlayPause?: boolean;
```

Comment:
This is not a disable option, this is a new feature we're offering. So maybe name it: providePlayPauseControl.

In this case, not providing the controls is the default. We only want to return the controls if the option specifically requests it

Reply:
got it, will push changes when have clear your other comment.